src/main/java/uk/gov/hmcts/darts/authentication/controller/AuthenticationExternalUserController.java

```diff
@@ -1,6 +1,8 @@
 package uk.gov.hmcts.darts.authentication.controller;
 
 import groovy.util.logging.Slf4j;
```

Comment:
Another different `Slf4j` import - we probably need to check them throughout?

Reply:
Same as above.  I think this groovy Slf4j was added with a manual logger at the very beginning of the development. Its cleaned now.